### PR TITLE
feat: SOF-1420 send schema design values to CasparCG

### PR DESCRIPTION
**Pull request by ianshade**
(Note: new ts-ignores will be gone when this is merged and released https://github.com/tv2/tv-automation-server-core/pull/249)

### CODE REVIEW

File: src/tv2_afvd_showstyle/helpers/pieces/graphicBackgroundLoop.ts [hunk 1]
```diff
@@ -1,75 +1,73 @@
...
 	parsedCue: CueDefinitionBackgroundLoop,
 	adlib?: boolean,
 	rank?: number
-) {
+): EvaluateCueResult {
+	const result = new EvaluateCueResult()
 	const start = (parsedCue.start ? calculateTime(parsedCue.start) : 0) ?? 0
 
 	if (parsedCue.target === 'DVE') {
+		const dveLoopGenerator = new DveLoopGenerator() // todo: where to instantiate it?
```
Comment (LindvedKrvang) [inline src/tv2_afvd_showstyle/helpers/pieces/graphicBackgroundLoop.ts:25]: Unfortunately, we don't have dependency injection, nor is this method inside a class. So for now, I think this is an okay place for it (not great, but okay).

If you want you could also utilize a Facade that would then handle the creation of the DveLoopGenerator and then this method you are in would not have to care about the instantiation. That responsibility would be handled by the Facade.

But I'm okay with instantiating here for now.

Comment (LindvedKrvang) [reply src/tv2_afvd_showstyle/helpers/pieces/graphicBackgroundLoop.ts:25]: You could also do something similar to how `GfxSchemaGenerator` is instantiated in `getRundown.ts`

**Review by LindvedKrvang — APPROVED** [review]

File: src/tv2_afvd_showstyle/helpers/pieces/evaluateCues.ts [hunk 2]
```diff
@@ -51,7 +51,8 @@ export async function EvaluateCues(
 			EvaluateCueGraphic,
 			EvaluateCueBackgroundLoop,
 			EvaluateCueGraphicDesign: EvaluateCueDesign,
-			EvaluateCueGraphicSchema,
+			EvaluateCueGraphicSchema: (c, p, partId, parsedCue) =>
```
Comment (KvelaGorrrrnio) [inline src/tv2_afvd_showstyle/helpers/pieces/evaluateCues.ts:54]: From this I have no idea what `c` and `p` are. Would be nice with describing names.

**Review by KvelaGorrrrnio — APPROVED** [review]
Looks good :rocket:

**Review by RasmusAlbrektsen — APPROVED** [review]

File: src/tv2_afvd_showstyle/helpers/pieces/evaluateCues.ts (hunk 2 above)
Comment (KvelaGorrrrnio) [reply src/tv2_afvd_showstyle/helpers/pieces/evaluateCues.ts]: `c` and `p` have been renamed to `cue` and `piece`.

**Pull request merged**